requirements/docs.in

```diff
@@ -1,13 +0,0 @@
--r base.in
-ipykernel
-ipympl
-ipython!=8.7.0  # Breaks syntax highlighting in Jupyter code cells.
-ipywidgets
-matplotlib
-nbsphinx
-plopp
-sphinx
-sphinx-autodoc-typehints==1.18.3  # later versions require sphinx >=5 which is incompatible with current sphinx-book-theme
-sphinx-book-theme==0.3.3
```

Comment:
Didn't you say you were using the new theme?

Reply:
Reverted after we postponed this in Scipp.